### DISCUSSION TITLE
All anchor links have active classes when using NuxtLink

### DISCUSSION THREAD

**Discussion by philipbradbury1** [nuxt/nuxt#22167] [Questions]
What using NuxtLink for a nav where all links are anchor links all the links have the active classes router-link-active router-link-exact-active. This is a problem when trying to create styles and accessibility. 

`
<template>
    <nav>
        <div>
            <NuxtLink @click="$emit('link-clicked')" :to="{ path: '/', hash:'#about'}">About</NuxtLink>
            <NuxtLink @click="$emit('link-clicked')" :to="{ path: '/', hash:'#featured'}">Projects</NuxtLink>
            <NuxtLink @click="$emit('link-clicked')" :to="{ path: '/', hash:'#work'}">Work</NuxtLink>
            <NuxtLink @click="$emit('link-clicked')" :to="{ path: '/', hash:'#contact'}">Contact</NuxtLink>
        </div>
    </nav>
</template>
`

**Comment by Jannchie** [comment]
This seems to be related to your question. 
https://router.vuejs.org/guide/migration/#removal-of-the-exact-prop-in-router-link

For some reason, "vue-router“ removed the API related to exact matching.
It mentions that:

> If you wish to customize this behavior, e.g. take into account the hash section, you should use the v-slot API to extend <router-link>.
